docs/Evidence/workflow/evidence-workflowConfig.json

```diff
@@ -0,0 +1,98 @@
+{
+  "BusinessServices": [
+    {
+      "tenantId": "kl",
+      "business": "evidence",
+      "businessService": "evidence-default",
+      "businessServiceSla": 518400000,
+      "states": [
+        {
+          "sla": 172800000,
+          "state": null,
+          "applicationStatus": null,
+          "docUploadRequired": true,
+          "isStartState": true,
+          "isTerminateState": false,
+          "isStateUpdatable": true,
+          "actions": [
+            {
+              "action": "TYPE_DEPOSITION",
+              "nextState": "PENDING_SIGNATURE",
+              "roles": [
+                "EVIDENCE_CREATOR"
+              ]
+            }
+          ]
+        },
+        {
+          "sla": 172800000,
+          "state": "PENDING_SIGNATURE",
+          "applicationStatus": "INWORKFLOW",
+          "docUploadRequired": true,
+          "isStartState": false,
+          "isTerminateState": false,
+          "isStateUpdatable": true,
+          "actions": [
+            {
+              "action": "E-SIGN",
+              "nextState": "PENDING_APPROVAL",
+              "roles": [
+                "DEPOSITION_ESIGN"
```

Comment:
This is still DEPOSITION_ESIGN. Should be EVIDENCE_ESIGNER.

Reply:
esign will be by EVIDENCE_CREATOR